toolchain/sem_ir/inst.h

```diff
@@ -265,9 +265,6 @@ class Inst : public Printable<Inst> {
   auto arg1() const -> int32_t { return arg1_; }
 
   // Returns arguments with their IdKind.
-  auto type_id_and_kind() const -> ArgAndKind {
-    return ArgAndKind(SemIR::IdKind::For<SemIR::TypeId>, type_id_.index);
-  }
```

Comment:
Should this comment be updated to drop mentions of type_id, or is this still used for that somewhere?
```
  // Associates an argument (usually arg0 or arg1, potentially type_id) with its
  // IdKind.
  class ArgAndKind {
```